index.html

```diff
@@ -1631,16 +1631,17 @@ <h3 id="applying">
           <code title="">related_applications</code> member
         </h3>
         <p>
+          A <dfn>related application</dfn> is an application provided by the
+          underlying application platform that has a relationship with the
+          web application <a>associated with a manifest</a>.
+        <p>
           The <dfn id=
           "member-related_applications"><code>related_applications</code></dfn>
-          member lists all the applications related to the web application.
-        </p>
-        <p>
-          The <code>related_applications</code> member is as an indication of a
-          relationship of the web application and the listed applications. This
-          relationship is unidirectional and unless a listed application claims
-          the same relationship, the <a>user agent</a> MUST NOT assume a
-          bi-directional endorsement.
+          member lists <a>related application</a>s and serves as an indication
+          of such a relationship between web application and <a>related
+          application</a>s. This relationship is unidirectional and unless a
+          listed application claims the same relationship, the <a>user agent</a>
```

Comment:
Also interested, although out of scope for the spec. Feel like suggesting a new feature to http://developer.android.com/guide/topics/manifest/manifest-intro.html and friends ?-)